### ISSUE TITLE
Failure to extract text from AMD GPU ISA docs

### ISSUE THREAD

**Issue by inequation** [jrmuizel/pdf-extract#37]
Frankly, I have no clue whether the problem lies in pdf-extract, or in one of its dependencies, please redirect me if this issue is misplaced.

For the public AMD GPU ISA documentation, such as:
https://developer.amd.com/wordpress/media/2012/12/AMD_Southern_Islands_Instruction_Set_Architecture.pdf
pdf-extract extracts blank pages. Other extractors, such as PyPDF2, extract the text just fine.

**Comment by jrmuizel** [comment]
That PDF is encrypted. I filed https://github.com/J-F-Liu/lopdf/issues/168 about adding support for RC4 encryption to lopdf.

**Comment by inequation** [comment]
Thank you! I certainly lack the context knowledge to diagnose this. :)

**Comment by saviour123** [comment]
@inequation Any update on this?

**Comment by inequation** [comment]
> @inequation Any update on this?

How should I know? I'm not the developer, just a user. :) I ended up extracting the text manually, as the formatting appears to confuse converters to the point where output is useless - columns in tables get all mixed up between rows.

**Comment by jwhear** [comment]
FYI, my implementation of RC4 decryption was just merged into lopdf: https://github.com/J-F-Liu/lopdf/pull/228, so this should now be unblocked.

**Comment by prscoelho** [comment]
@jrmuizel any chance of picking this up now that lopdf supports decryption? Or, would you accept a pr for this?
